.github/workflows/action.yml

```diff
@@ -0,0 +1,14 @@
+name: Check Markdown links
+
+on: push
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
Can you pin a specific commit? That reduces the risk of a malicious update.

Reply:
Done, pinned to current version ([1.0.12](https://github.com/gaurav-nelson/github-action-markdown-link-check/commit/0fe4911067fa322422f325b002d2038ba5602170))